clang/test/CodeGen/builtins-elementwise-math.c

```diff
@@ -666,40 +666,16 @@ void test_builtin_elementwise_log2(float f1, float f2, double d1, double d2,
   vf2 = __builtin_elementwise_log2(vf1);
 }
 
-void test_builtin_elementwise_popcount(si8 vi1, si8 vi2,
-                                  long long int i1, long long int i2, short si,
-                                  _BitInt(31) bi1, _BitInt(31) bi2) {
-
-  
-  // CHECK:      [[I1:%.+]] = load i64, ptr %i1.addr, align 8
```

Comment:
Did the IR change for the whole function?
Could you please reflect that in the new checks instead of simply removing the old checks?